runtime/testy/studentCode.py

```diff
@@ -8,10 +8,35 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  print("setup")
+  devices = Robot._getValue("hibike", "devices")
```

Comment:
I think this is still the wrong function name (should be `_getSMValue`).
On another topic, it'd probably be good to standardize our function naming convention to either be under_scores (my vote) or camelCase, as it's currently inconsistent across the codebase.